src/character.h

```diff
@@ -91,6 +91,17 @@ class Character : public Creature
         {
             return -2 - position;
         }
+
+        // checks to see if an item is worn
+        bool is_worn(const item &thing) const
+        {
+            for(const auto &elem : worn) {
```

Comment:
Most of C++'s std library is a learn by use basis for me. I come from C and Assembler primarily, this is really the first time using C++ in a project. :-)